docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Migrating from Expo Firebase to React Native Firebase
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+The `expo-firebase-analytics` and `expo-firebase-recaptcha` packages allowed the use of some Firebase features on Classic Builds, Expo Go, and Expo for Web, but developers often encountered version conflicts when trying to use them alongside the rest of the Firebase suite. With EAS Build and Development Builds, it is much easier to use [React Native Firebase](https://rnfirebase.io/) in your entire development workflow, so these packages have been deprecated.
+
+In this guide, we will migrate from the Expo Firebase packages to React Native Firebase by:
+- Replacing the Expo Firebase packages with their React Native Firebase equivalents
+- Replacing the use of Expo Go for testing with a development build customized to your project's needs
+- Testing the updated code on the development build
```

Comment:
@tcdavis what's our canonical language for referring to / branding Dev Client? We should use the same terminology here both to piggy back on existing familiarity but also to buttress the DevClientNameHere* brand? 

(*Sorry, I looked on https://expo.dev/tools to see what we're using but realized it hasn't joined that fam yet)

Reply:
+1 to this question! I do see "Dev Clients" in the glossary (https://docs.expo.dev/workflow/glossary-of-terms/#dev-clients) and we call it that on Discord, etc, but searching "Development Builds" gets you to best place to get the whole rundown on how to build one.